docs/device-assignment.md

```diff
@@ -0,0 +1,130 @@
+# Virtual Machine Device Assignment
+
+## Scope
+
+For the first version of the document, the scope is reduced PCI device assignment.
+
+## Introduction
+
+Device assignment is a feature that allows virtual machines (VMs) to take advantage of hardware capabilities and performance available on the hypervisor (host/node). Unlike traditional VM resources such as virtual CPUs (vCPUs) or memory, assignment-capable hardware devices are limited resource that cannot be overcommitted.
+
+### Device Assignment in Kubernetes
```

Comment:
Please provide some links to the relevant K8s docs/prs.

Reply:
Done, embedded below.